package.xml

```diff
@@ -36,6 +36,8 @@
   <depend>visualization_msgs</depend>
 
   <build_depend>assimp-dev</build_depend>
+  <build_depend>eigen</build_depend>
+  <build_depend>libfcl-dev</build_depend>
```

Comment:
`<exec_depend>libfcl-dev</exec_depend>` is also needed.

Reply:
Having both the `<build_depend>libfcl-dev</build_depend>` and `<exec_depend>libfcl-dev</exec_depend>` caused the CI to fail, is it enough to have it in either exec_depend or just `<depend>...</depend>`

Reply:
This is wrong. `<exec_depend>libfcl-dev` has to be there.

Reply:
Or, even better, `<exec_depend>fcl</exec_depend>` (but wait until https://github.com/ros/rosdistro/pull/43530  is merged).